KProcessHacker/verify.c

```diff
@@ -377,48 +390,50 @@ NTSTATUS KpiVerifyProcessSections(
 
     {
         status = STATUS_ACCESS_DENIED;
-        goto Exit;
+        goto CleanupExit;
     }
 
-    bytes = ExAllocatePoolWithTag(PagedPool, FILE_BUFFER_SIZE, 'pvpK');
-    if (!bytes)
+    if (!(bytes = ExAllocatePoolZero(PagedPool, FILE_BUFFER_SIZE, 'pvpK')))
     {
         status = STATUS_INSUFFICIENT_RESOURCES;
-        goto Exit;
+        goto CleanupExit;
     }
 
     remainingBytes = inspectBytes;
     offset.QuadPart = textSection->PointerToRawData;
-    while (remainingBytes > 0)
+
+    while (remainingBytes != 0)
```

Comment:
diff didn't pick up on `>` changing to `!=` - probably due to the new line, making it harder to review